packages/user-mgnt/src/config/routes.ts

```diff
@@ -340,11 +340,11 @@ export const getRoutes: () => Hapi.ServerRoute[] = () => {
     },
     {
       method: 'GET',
-      path: '/users/{userId}/avatar',
+      path: '/practitioners/{practitionerId}/avatar',
```

Comment:
This could be bit of an abstraction leak from FHIR/Hearth to our user-mgnt. Do you have an idea how the route should be handled.. 🤔 Maybe like /users/avatar?practitionerId=xyz

Reply:
Why I changed this, is that AssignmentData in gateway is the only thing solving the user's avatar. https://github.com/opencrvs/opencrvs-core/blob/befbfd099eea5b4f05db4b2acef50953dfc1c445/packages/gateway/src/features/search/type-resolvers.ts#L464

When streaming the records to ElasticSearch though, I wanted to remove a user call to user-mgnt to find the id of a practitioner. I'm now considering if the above link should first "getPractitionerIdByUserId" (probably such a method needs to be implemented) and then it calls user-mgnt with the proper user id?

Reply:
Okay now I now why you did what you did. The search user endpoint should support practitioner id. maybe we can use that?